<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="77895" started-at="2021-02-10T08:58:19Z" finished-at="2021-02-10T08:59:37Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="77895" started-at="2021-02-10T08:58:19Z" finished-at="2021-02-10T08:59:37Z">
      <class name="TestCaseExecution.LoginTest">
        <test-method status="PASS" signature="setextentreport()[pri:0, instance:TestCaseExecution.LoginTest@69d9c55]" name="setextentreport" is-config="true" duration-ms="593" started-at="2021-02-10T08:58:19Z" finished-at="2021-02-10T08:58:19Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setextentreport -->
        <test-method status="PASS" signature="openbrowser()[pri:0, instance:TestCaseExecution.LoginTest@69d9c55]" name="openbrowser" is-config="true" duration-ms="36667" started-at="2021-02-10T08:58:19Z" finished-at="2021-02-10T08:58:56Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openbrowser -->
        <test-method status="FAIL" signature="TestCase_One()[pri:0, instance:TestCaseExecution.LoginTest@69d9c55]" name="TestCase_One" duration-ms="10809" started-at="2021-02-10T08:58:56Z" finished-at="2021-02-10T08:59:07Z">
          <exception class="junit.framework.AssertionFailedError">
            <full-stacktrace>
              <![CDATA[junit.framework.AssertionFailedError
	at junit.framework.Assert.fail(Assert.java:47)
	at junit.framework.Assert.assertTrue(Assert.java:20)
	at junit.framework.Assert.assertFalse(Assert.java:34)
	at junit.framework.Assert.assertFalse(Assert.java:41)
	at TestCaseExecution.LoginTest.TestCase_One(LoginTest.java:80)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception> <!-- junit.framework.AssertionFailedError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TestCase_One -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:TestCaseExecution.LoginTest@69d9c55]" name="tearDown" is-config="true" duration-ms="28123" started-at="2021-02-10T08:59:07Z" finished-at="2021-02-10T08:59:35Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=TestCase_One status=FAILURE method=LoginTest.TestCase_One()[pri:0, instance:TestCaseExecution.LoginTest@69d9c55] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="endreport()[pri:0, instance:TestCaseExecution.LoginTest@69d9c55]" name="endreport" is-config="true" duration-ms="1537" started-at="2021-02-10T08:59:35Z" finished-at="2021-02-10T08:59:37Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- endreport -->
      </class> <!-- TestCaseExecution.LoginTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
